README.md

```diff
@@ -149,6 +149,14 @@ OS memory provider supports two types of memory mappings (set by the `visibility
 
 IPC API requires the `UMF_MEM_MAP_SHARED` memory `visibility` mode
 (`UMF_RESULT_ERROR_INVALID_ARGUMENT` is returned otherwise).
+IPC API uses the file descriptor duplication. It requires using `pidfd_getfd(2)` to obtain
```

Comment:
Iif you want to start this from a new paragraph you should add an empty line before.

right now it's all written together:
![image](https://github.com/user-attachments/assets/19bede1f-f51c-4a31-8e4a-dc01786e0c76)

Reply:
Done